code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -178,6 +178,20 @@
 		M.update_inv_l_hand()
 		M.update_inv_r_hand()
 
+/obj/item/reagent_containers/syringe/Crossed(mob/living/carbon/human/H, oldloc)
+	if(H.reagents && !(HAS_TRAIT(H,TRAIT_PIERCEIMMUNE) || ismachineperson(H) || H.floating || H.flying || H.buckled || H.shoes || (H.wear_suit && (H.wear_suit.body_parts_covered & FEET)) || (H.w_uniform && (H.w_uniform.body_parts_covered & FEET))))
```

Comment:
There are a lot of negative checks here. Could you please make this an early return that isn't negated?
Something (roughly) like:
```
if(!H.reagents)
    return
if(HAS_TRAIT(...) || ismachineperson(H) || ....)
    return
H.visible_message(....)
...
```

Would make it a lot easier to understand.